# Modeling Hepatoblastoma: Identification of Distinct Tumor Cell Populations and Key Genetic Mechanisms through Single Cell Sequencing.

Single cell RNAseq was used to further characterize the HB and PDX tumor compared to background liver to determine integral pathways and cell signatures in HB utilizing HB17, HB30, and HB53 samples (the second passage (F2) was used for available PDX samples). After preprocessing and quality control, we collected 67,111 cells across seven samples, including three tumors, two background livers and two PDX tumors.

Here're codes which were used to conduct analysis and draw figures in this paper: Modeling Hepatoblastoma: [Identification of Distinct Tumor Cell Populations and Key Genetic Mechanisms through Single Cell Sequencing.] (https://github.com/KANG-BIOINFO/scRNA-seq_Hepatoblastoma/edit/main/README.md)

Supplementary Figures: Analysis of single nuclei RNA in each sample group and visualization;</br>
Figure 3: Systemic analysis of integrated cells from all sample groups;
Figure 4: Scatter plot of gene expression levels across sample groups; Volcano plots of DEGs; Cell cycle scores across cell types.
Figure 5: Subclustering of tumor cells; Visualization of tumor signatures.
Figure 6: RNA velocity of tumor cells;
Figure 7: Interaction inference using CellChat.
